securetea/core.py

```diff
@@ -39,13 +40,18 @@ def __init__(self):
         args = get_args()
         credentials = configurations.SecureTeaConf()
         if(args.twitter_api_key and args.twitter_api_secret_key and args.twitter_access_token and
-                args.twitter_access_token_secret):
+                args.twitter_access_token_secret and args.telegram_token and args.telegram_user_id):
```

Comment:
We only need to make one integration mandatory. All other integration should be optional. If the user configured at least one integration we allow them to run the application. 
So, Make sure not to force multiple integrations.

Reply:
Okay, I'll update this.